models/asset.js

```diff
@@ -4,11 +4,21 @@ var Schema = mongoose.Schema;
 var assetSchema = Schema({
 	name:String,
 
-	content:String,
+	category:String,
 
-	createDate:Date,
+	version:{
+		no:Number,
+		date:Date
```

Comment:
1. A string? not number?
2. a version date is the date for the specific version